analyzers/tests/ITs.JsonParser.Test/Sarif.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace ITs.JsonParser.Test;
+
+internal static class Sarif
```

Comment:
None of this should be `internal`. Nobody will ever reference this project from outside